tests/integration-tests/tests/storage/test_fsx_lustre.py

```diff
@@ -525,6 +525,7 @@ def _create_fsx_lustre_volume_ids(num_existing_fsx_lustre, fsx_factory, import_p
             DeploymentType="PERSISTENT_1",
             PerUnitStorageThroughput=200,
         ),
+        FileSystemTypeVersion="2.12",
```

Comment:
I'm not sure we should specify the version type.  Can we not let it use the default?

Reply:
Default from CloudFormation is 2.10. Default from AWS console is 2.12. So we have to specify the version